po/de.po

```diff
@@ -941,23 +938,20 @@ msgstr ""
 "<subtask>7@1</subtask>\n"
 "<subtask>8@1</subtask>\n"
 "\n"
-"Genauso können sie die GTG Dokumentation durchsuchen, indem sie F1 drücken, "
+"Genauso können Sie die GTG Dokumentation durchsuchen, indem sie F1 drücken, "
 "oder das Hilfe Menü verwenden.\n"
 "\n"
 "Wir hoffen aufrichtig, dass sie Freude an der Benutzung von GTG haben, und "
-"bedanken uns, dass sie es ausprobieren! Bitte senden sie uns Fehlerberichte "
-"und Verbesserungsvorschläge über die Webseite: https://bugs.launchpad.net/"
-"gtg/+filebug Wenn sie Tipps erhalten wollen, wie man GTG verwendet oder wenn "
-"sie über die neusten Verbesserungen auf dem laufenden bleiben wollen, dann "
-"besuchen sie unseren Blog auf http://gtg.fritalk.com\n"
+"bedanken uns, dass Sie es ausprobieren! Auf unserer Webseite https://wiki."
+"gnome.org/Apps/GTG/ finden Sie mehr Informationen über das Projekt und wie "
+"Sie mitwirken können.\n"
 "\n"
-"Das GTG Team"
+"— Das GTG Team"
 
 #. Task 1@1: Learn to use subtasks
 #: GTG/core/firstrun_tasks.py:109
-#, fuzzy
 msgid "Learn How to Use Subtasks"
-msgstr "Lerne sie wie man Teilaufgaben verwendet"
+msgstr "Lerne, wie man Teilaufgaben verwendet"
```

Comment:
Thanks for the wiki page!
I'll use your suggestion, but I think I initially went form of non-courtesy because it is used as task titles and being polite in task titles seemed weird to me. (Although I now notice that the text inside is also in form of courtesy...)

Reply:
Maybe it doesn't have to be personal and we could say "Lernen, Teilaufgaben zu verwenden". I'm not sure though.

Reply:
It should at least be consistent with the text inside.

Reply:
I actually like your suggestion, I'll look into it in the next round of translation (want to get this merged before more work pile up with new strings)